cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -587,17 +588,17 @@ protected void dynamicUpdateBrokerConfig(int podId, Admin ac, KafkaBrokerConfigu
             });
         await(Util.kafkaFutureToVertxFuture(reconciliation, vertx, brokerLoggingConfigFuture), 30, TimeUnit.SECONDS,
             error -> {
-                LOGGER.errorCr(reconciliation, "Error performing dynamic logging update for pod {}", podId, error);
-                return new ForceableProblem("Error performing dynamic logging update for pod " + podId, error);
+                LOGGER.errorCr(reconciliation, "Error performing dynamic logging update for pod {}", podRef, error);
```

Comment:
Again, it would be better to stick to a single verb (i.e. not adding "performing" to the vocabulary). Could we say something like "Error updating logging config for pod" and "Error updating broker config for pod"?

Reply:
done